README.md

```diff
@@ -1,6 +1,8 @@
 # PyoMyo
 Python module for the Thalmic Labs Myo armband. 
 
+## Making a change here to demonstrate how Sema works. 
```

Comment:
well i think you are doing a good job with this but a thing you m
Many developers feel an irresistible urge to start coding immediately. Although it's an understandable approach, it's not the best one. Making an application is a complex process and you can seldom skip a pha...

📄 [Ideamotive](https://www.ideamotive.co/blog/best-practices-in-ios-app-development)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good